examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -0,0 +1,140 @@
+"""
+========================================
+Release Highlights for scikit-learn 0.23
+========================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 0.23! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. For an exhaustive list of
+all the changes, please refer to the :ref:`release notes <changes_0_23>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install scikit-learn
+"""
+
+# flake8: noqa
+
+##############################################################################
+# Generalized Linear Models
+# -------------------------
+# Long-awaited Generalized Linear Models with non-normal loss functions are now
+# available. In particular, three new regressors were implemented:
+# :class:`~sklearn.linear_model.PoissonRegressor`,
+# :class:`~sklearn.linear_model.GammaRegressor`, and
+# :class:`~sklearn.linear_model.TweedieRegressor`. The Poisson regressor can be
+# used to model positive integer counts, or relative frequencies. Read more in
+# the :ref:`User Guide <Generalized_linear_regression>`.
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.linear_model import PoissonRegressor
+
+n_samples, n_features = 1000, 20
+rng = np.random.RandomState(0)
+X = rng.randn(n_samples, n_features)
+y = ((10 + X[:, 5]) * 10).astype(int)  # easy target correlated with X[:, 5]
```

Comment:
The following gives a problem with a slight majority of zeros that is partially solved by the generalized linear and the HGBRT models (score in the ~0.3, depending on the seed).

```suggestion
# positive integer target correlated with X[:, 5] with many zeros:
y = rng.poisson(lam=np.exp(X[:, 5]) / 2)
```